components/ws-daemon/pkg/content/service.go

```diff
@@ -627,7 +658,7 @@ func (s *WorkspaceService) uploadWorkspaceLogs(ctx context.Context, sess *sessio
 			return
 		})
 		if err != nil {
-			return xerrors.Errorf("cannot upload workspace content: %w", err)
+			return fmt.Errorf("cannot upload workspace content: %w", err)
```

Comment:
Why switch to `fmt` here?
Doesn't line up with our [style guide](https://github.com/gitpod-io/go-style-guide/blob/master/style.md#errors)

Reply:
Fixed